.github/workflows/test-and-deploy.yml

```diff
@@ -35,18 +36,32 @@ jobs:
       - name: Run Tests
         run: make test-docker
 
+  deploy:
+    name: Deploy
+    if: success() && github.ref_type == 'tag'
+    needs: [ test ]
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout sendgrid-go
+        uses: actions/checkout@v2
+
+      - name: Create GitHub Release
+        uses: sendgrid/dx-automator/actions/release@main
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Might take this out so we can make all the GitHub release changes to the SendGrid repos at the same time. This simplifies the changes needed to librarian.

Reply:
Ack. I guess we can then merge this PR later since all the `deploy` does is create a release.